specification/indices/put_settings/IndicesPutSettingsRequest.ts

```diff
@@ -105,6 +105,13 @@ export interface Request extends RequestBase {
      * @server_default false
      */
     preserve_existing?: boolean
+    /**
+     * Whether to close and reopen the index to apply non-dynamic settings.
+     * If set to `true` the indices to which the settings are being applied
+     * will be closed temporarily and then reopened in order to apply the changes.
+     * @server_default false
+     */
+    reopen?: boolean
```

Comment:
This parameter is read in https://github.com/elastic/elasticsearch/blob/d9e751602dc4a874ef381797844f80044889fbef/server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestUpdateSettingsAction.java#L52 since [8.12.0](https://github.com/elastic/elasticsearch/pull/101723).